docs/Qna.md

```diff
@@ -0,0 +1,23 @@
+# Questions and Answers
+
+## What is the `db` module
```

Comment:
These titles should have question marks.

Reply:
lint doesn't allow question marks in titles so I'm gonna rephrase the titles into statements.